core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/source/Sources.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2018 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.comparators.source;
+
+
+import com.cognifide.aet.job.api.comparator.ComparatorProperties;
+import com.cognifide.aet.job.api.datafilter.DataFilterJob;
+import com.cognifide.aet.job.api.exceptions.ProcessingException;
+import com.cognifide.aet.vs.ArtifactsDAO;
+import java.io.IOException;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class Sources {
```

Comment:
Could we have short descrption (JAVADOC) on what this class does?

Reply:
Done